addons/binding/org.openhab.binding.rfxcom/README.md

```diff
@@ -67,16 +67,16 @@ If you have any problems with JD2XX or you don't want to disable FTDI driver on
 
 To do that via the PaperUI, manually add the generic RFXCOM device named `RFXCOM USB Transceiver`, with the description "This is universal RFXCOM transceiver bridge for manual configuration purposes".
 You will need to specify at least the serial port which has been assigned to the RFXCOM (see notes above).
-You may also need to change permissions on the serial port to allow openhab to access it and you may need to tell the java libraries about it by adding a line to &lt;openhab root&gt;/conf/environment such as:
+You may also need to change permissions on the serial port to allow openhab to access it and you may need to tell the java libraries about it by adding a line to etc/default/openhab2 such as:
```

Comment:
This change is not generally correct as this path is only valid for apt-based installations, but not for any other.

Reply:
My suggestion is to completely remove this sentense here (and the EXTRA_JAVA_OPTS example) and merely add a short pointer to https://www.openhab.org/docs/administration/serial.html here.